htsinfer/infer_read_orientation.py

```diff
@@ -22,5 +21,45 @@ def infer(
         LIBTYPE string according to Salmon documentation, cf.
         https://salmon.readthedocs.io/en/latest/library_type.html
     """
+    #create .tmp directory in "inputDir" and retuns .tmp path
+    tmpDir = mkTmpDir(inputDir)
+    
+    # call functions of issues #25, 26, 27 and 34
+    
+    # delete .tmp
+    rmTmpDir(tmpDir)
+        
     # implement logic
     return "U"
+    
+
+def mkTmpDir(inputDir):
+    """Function to create a temporary directory for the storage of 
```

Comment:
Please always make sure to describe input params, the return value and potentially raised errors in the docstring and follow the Google-style convention to do so. More examples for this are in the `infer_single_paired.py` module.